src/components/layout/Header.jsx

```diff
@@ -0,0 +1,19 @@
+import { useUser } from '../../context/UserContext';
+
+const Header = () => {
+  const { user } = useUser();
+
+  return (
+    <header>
+      {user ? (
+        <p>
+          You are signing the guestbook as <span>{user}</span>
+        </p>
+      ) : (
+        <p>Please sign in!</p>
```

Comment:
Ahh, so this doesn't appear to be working totally. When I sign the guestbook, it always says "Please sign in!" rather than switching to saying "You are signing the guestbook as <name>".